packages/api-request/src/middlewares/test/namespace-endpoint.js

```diff
@@ -0,0 +1,18 @@
+import namespaceEndpointMiddleware from '../namespace-endpoint';
+
+describe( 'Namespace & Endpoint middleware', () => {
+	it( 'should concat the endpoint and namespace into a path property', () => {
+		const requestOptions = {
+			method: 'GET',
+			namespace: '/wp/v2',
+			endpoint: '/posts',
+		};
+		const callback = ( options ) => {
+			expect( options.path ).toBe( 'wp/v2/posts' );
+			expect( options.namespace ).toBeUndefined();
+			expect( options.endpoint ).toBeUndefined();
+		};
+
+		namespaceEndpointMiddleware( requestOptions, callback );
```

Comment:
We should add [expect.hasAssertions](https://facebook.github.io/jest/docs/en/expect.html#expecthasassertions) call to make sure that `callback` was executed.